### PR TITLE
FI-1100 Migrate USCore 311 code generation update to Community

### PR DESCRIPTION
**Pull request by yunwwang**
This PR includes
* Update code generator and templates for US Core IG v3.1.1 
* Remove testing for US Core IG 3.1.0

This PR does not include:
* Validator update for US Core IG. Validate update is include in another PR

### CODE REVIEW

File: generator/uscore/metadata_extractor.rb [hunk 1]
```diff
@@ -556,6 +549,10 @@ def set_first_search(sequence, params)
         sequence[:searches].delete(search)
         sequence[:searches].unshift(search)
       end
+
+      def capitalize_first_letter(str)
```
Comment (Jammjammjamm) [inline generator/uscore/metadata_extractor.rb:553]: We should get rid of this method and go back to using `String#upcase_first`, which we get from `ActiveSupport`.

**Review by Jammjammjamm — COMMENTED** [review]
When you say the validator update will be in a separate PR, what do you mean by that? The changes in these PRs?
- https://github.com/onc-healthit/inferno-program/pull/212
- https://github.com/onc-healthit/inferno-program/pull/214

**Comment by yunwwang** [comment]
This validator PR: https://github.com/onc-healthit/inferno/pull/502

File: generator/uscore/metadata_extractor.rb (hunk 1 above)
Comment (yunwwang) [reply generator/uscore/metadata_extractor.rb:553]: changes are reversed. Why do we have to use different pattern in Community?

**Comment by arscan** [comment]
Replaced by #526

**Pull request closed without merging**